src/pages/components/table.mdx

```diff
@@ -0,0 +1,210 @@
+    <ul>
+      <li>To organize and display data with logical relationships</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To structure content on a page, use the [Grid](/components/grid) component instead
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a Table component.
+
+import Default from '../../examples/table/TableDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/table/TableDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Grouped rows
+
+Grouped rows are for additional categorization.
+
+import GroupedRows from '../../examples/table/TableGroupedRows.tsx';
+import GroupedRowsCode from '!!raw-loader!../../examples/table/TableGroupedRows.tsx';
+
+<CodeExample code={GroupedRowsCode}>
+  <GroupedRows />
+</CodeExample>
+
+### Overflow
+
+Overflow menus leverage the Dropdown and Menu component.
+
+import Overflow from '../../examples/table/TableOverflow.tsx';
+import OverflowCode from '!!raw-loader!../../examples/table/TableOverflow.tsx';
+
+<CodeExample code={OverflowCode}>
+  <Overflow />
+</CodeExample>
+
+### Pagination
+
+Table pagination works in combination with the [Pagination](/components/pagination) component.
+
+import Pagination from '../../examples/table/TablePagination.tsx';
+import PaginationCode from '!!raw-loader!../../examples/table/TablePagination.tsx';
+
+<CodeExample code={PaginationCode}>
+  <Pagination />
+</CodeExample>
+
+### Scroll
+
+To ensure that your table is read correctly by assistive technologies follow the
+[W3C Grid accessibility
+pattern](https://www.w3.org/TR/wai-aria-practices-1.1/examples/grid/dataGrids.html).
+
+import Scroll from '../../examples/table/TableScroll.tsx';
+import ScrollCode from '!!raw-loader!../../examples/table/TableScroll.tsx';
+
+<CodeExample code={ScrollCode}>
+  <Scroll />
+</CodeExample>
+
+### Selection
+
+Individual rows within a table can be selected for bulk actions.
+
+import SelectRows from '../../examples/table/TableSelectRows.tsx';
+import SelectRowsCode from '!!raw-loader!../../examples/table/TableSelectRows.tsx';
+
+<CodeExample code={SelectRowsCode}>
+  <SelectRows />
+</CodeExample>
+
+### Size
+
+Table row sizes comes in small, medium, and large. The default size is medium.
+
+import Size from '../../examples/table/TableSize.tsx';
+import SizeCode from '!!raw-loader!../../examples/table/TableSize.tsx';
+
+<CodeExample code={SizeCode}>
+  <Size />
+</CodeExample>
+
+### Sort
+
+Columns can be sorted by selecting table headers.
+
+import Sort from '../../examples/table/TableSort.tsx';
+import SortCode from '!!raw-loader!../../examples/table/TableSort.tsx';
+
+<CodeExample code={SortCode}>
+  <Sort />
+</CodeExample>
+
+### Striped rows
+
+The background color of rows can alternate to help delineate content when
+scanning long rows. Striped and grouped rows shouldn’t be active at the same
+time.
```

Comment:
Couple things here:
1. First sentence uses passive tense. Maybe instead: 

> You can alternate row background color to help make long rows easier to scan.

2. "shouldn't be active" feels a bit jargony… maybe change "active" to "used"

> Don't use striped and group rows at the same time.